samples/csharp_dotnetcore/02.echo-with-counter/Startup.cs

```diff
@@ -62,7 +63,10 @@ public void ConfigureServices(IServiceCollection services)
 
                 var secretKey = Configuration.GetSection("botFileSecret")?.Value;
                 var botFilePath = Configuration.GetSection("botFilePath")?.Value;
-
+                if (!File.Exists(botFilePath))
+                {
+                    throw new FileNotFoundException($"The .bot configuration file was not found. botFilePath: {botFilePath}");
```

Comment:
Looking at the code below, it appears if the botFilePath is not set (aka: set to null), then the code will default to ".\Botconfiguration.bot". This looks deliberate in the code below. 

I don't think this is a change that's needed. 

Now - a change that seems like a good idea would be to add the botfilePath (or default file name) to the exception message that's thrown on line 84.

Reply:
I agree with adding the filepath to that location, but this check is for if the botFilePath (either the one thats set or the default one) does not actually refer to a file that exists, currently this throws a "could not find file" error that isn't caught (seen in screenshot attached below from issue #730), but the bot still won't work. 

It also appears that this particular sample is an exception with regard to error handling, as the logic around line 84 does not exist in most of the other sample bots, so this would cover the use case in those bots as well if its approved for putting everywhere.

https://user-images.githubusercontent.com/23645105/47218657-e53d2900-d382-11e8-92cd-fc1f990fa92a.png